src/custom-transformations/settlement-report-params.js

```diff
@@ -0,0 +1,22 @@
+const getSettlementReportParams = (msg) => {
+  const reportingParams = {}
+
```

Comment:
I think this function can be put inside `EventProcessorService` class as a static method.

Reply:
I thought of implementing these as nodejs modules instead of classes as we are not going to create multiple instances.